test/e2e_node/container_lifecycle_test.go

```diff
@@ -3280,7 +3280,7 @@ var _ = SIGDescribe(nodefeature.SidecarContainers, "Containers Lifecycle", func(
 				ps3Last, err := results.TimeOfLastLoop(prefixedName(PreStopPrefix, restartableInit3))
 				framework.ExpectNoError(err)
 
-				const simulToleration = 500 // milliseconds
+				const simulToleration = 1000 // milliseconds
 				// should all end together since they loop infinitely and exceed their grace period
 				gomega.Expect(ps1Last-ps2Last).To(gomega.BeNumerically("~", 0, simulToleration),
 					fmt.Sprintf("expected PostStart 1 & PostStart 2 to be killed at the same time, got %s", results))
```

Comment:
Could we also fix `PostStart` to `PreStop` in the messages?

Reply:
I will open a new Pull Request PR to fix it. Although the fix is quite minor, we still have to worry about the possibility that the current PR might be reverted, which would cause this fix to be rolled back as well.